### DISCUSSION TITLE
How to call Apply script to documents from beginning of time (Reset) from UpdateEtlOperation

### DISCUSSION THREAD

**Discussion by ItsRobbAllen** [ravendb/ravendb#16582] [Q&A] [answered]
I have automated ETLs set up using `AddEtlOperations` and `UpdateEtlOperations` and it works fine. However, when I call `UpdateEtlOperations`, I'd like to set whatever flag that causes the ETL to start over from the beginning of time.

How do I set this?

**Comment by garayx** [comment, marked as answer]
Hi,

Did you see ResetEltOperation https://ravendb.net/docs/article-page/4.2/csharp/client-api/operations/maintenance/etl/reset-etl ?

See usage in our tests - https://github.com/ravendb/ravendb/blob/v4.2/test/SlowTests/Server/Documents/ETL/RavenDB_8866.cs

**Reply by ItsRobbAllen** [reply]
Sorry for the late reply. That should work, I'll just need to make 2 separate calls.
